AaronLiuIsCool/friend-school
# Integrate nativebase

# What is this PR
The goal of this PR is to get the following stuff into our code repo.
- Redux
- Nativebase
- Some boiler template + Side Drawer

### What I did
1. Install and Link Native Base
http://nativebase.io/docs/v2.0.0/getting-started

2. Copied files from the starter kit
https://github.com/start-react/native-starter-kit

3. Put the old Facebook login component into new place

### QA
I wouldn't recommend to review the changes. Most of the work is just installing and moving stuff around. The only actual work I did in last commit.
Maybe just pull it down and try it out and see how it works.
I will probably follow-up a cleaning PR to clear some things out.

- [x] QA'd by me
- [ ] QA'd by reviewer
